wram.asm

```diff
@@ -1361,37 +1361,68 @@ wMenuDataBank:: db
 wMenuDataHeaderEnd::
 
 wMenuData2::
-UNION ; cf91
 ; MenuData2
+
 wMenuData2Flags:: ; cf91
```

Comment:
Can this be split into wMenuData2Flags, wMenuData2_2DMenuFlags, and wMenuData2_ScrollingMenuFlags in those unions? Then the comments about different bit meanings would also go in their respective sections, engine/scrolling_menu.asm can use wMenuData2_ScrollingMenuFlags, etc.

Reply:
No, because some flags and code checking those flags are used for several different menus. See, for example, `GetMenuTextStartCoord`, which checks bit 6 and 7 of the flags, and is used by both `PlaceVerticalMenuItems` and `Place2DMenuItemStrings`.